djstripe/models/billing.py

```diff
@@ -1277,6 +1305,27 @@ class Subscription(StripeModel):
         "subscription. This value will be `null` for subscriptions where "
         "`billing=charge_automatically`.",
     )
+    default_payment_method = models.ForeignKey(
+        "PaymentMethod",
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
+        help_text="ID of the default payment method for the subscription. "
+        "It must belong to the customer associated with the subscription. "
+        "If not set, invoices will use the default payment method in the "
+        "customer’s invoice settings.",
+    )
+    default_source = PaymentMethodForeignKey(
+        on_delete=models.SET_NULL,
+        null=True,
+        blank=True,
+        related_name="subscriptions",
+        help_text="ID of the default payment source for the subscription. "
```

Comment:
Minor tweak - I'd remove "ID of the" from the help text copied from Stripe, since in our case it's an FK to an object.

Eg "Default payment source for the ..." instead.